### PR TITLE
Viewbookings

### PR DESCRIPTION
**Pull request by eninodel**
Added the ability for users to view the bookings they made

### CODE REVIEW

File: GearRent/ViewControllers/BookingTableViewCell.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//
+
+#import <UIKit/UIKit.h>
+#import "../Models/Reservation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface BookingTableViewCell : UITableViewCell
+
+@property (strong, nonatomic)Reservation *reservation;
```
Comment (alonbee) [inline GearRent/ViewControllers/BookingTableViewCell.h:15]: nit: space before Reservation

**Review by alonbee — APPROVED** [review]
looks good overall! accept to unblock

File: GearRent/ViewControllers/ReservationTableViewCell.m [hunk 2]
```diff
@@ -2,21 +2,20 @@
...
 //  GearRent
 //
-//  Created by Edwin Delgado on 7/15/22.
+//  Created by Edwin Delgado on 7/14/22.
 //
 
 #import "ReservationTableViewCell.h"
 
 @interface ReservationTableViewCell()
+@property (weak, nonatomic) IBOutlet UILabel *leaseeLabel;
```
Comment (alonbee) [inline GearRent/ViewControllers/ReservationTableViewCell.m:11]: let's use strong for these

**Review by alonbee — APPROVED** [review]

**Review by alonbee — APPROVED** [review]

File: GearRent/ViewControllers/BookingsViewController.m [hunk 3]
```diff
@@ -21,8 +23,16 @@ - (void)viewDidLoad {
...
+    [self fetchData];
+}
+
+- (void)fetchData{
     PFQuery *query = [PFQuery queryWithClassName:@"Reservation"];
     [query whereKey:@"leaseeId" equalTo:[[PFUser currentUser] objectId]];
+    [query includeKey:@"dates"];
     [query findObjectsInBackgroundWithBlock:^(NSArray * _Nullable objects, NSError * _Nullable error) {
         if(error == nil){
             self.reservations = objects;
```
[the remark this answers is not in the document]
Comment (eninodel) [reply GearRent/ViewControllers/BookingsViewController.m:38]: that makes sense

File: GearRent/ViewControllers/ReservationTableViewCell.m (hunk 2 above)
Comment (eninodel) [reply GearRent/ViewControllers/ReservationTableViewCell.m:11]: Will do in large cleanup PR

**Pull request merged**